docs/dictionary/command/send.lcdoc

```diff
@@ -38,21 +38,21 @@ message: An expression that evaluates to a message name, possibly including para
 object: Any object reference. If you don't specify an object, the message is set to the object whose handler contains the send command. If you specify a time, you must also specify an object.
 time (integer): If you don't specify a unit of time, the default of ticks is used.
 
-The result: If you specify a <time>, the <message> is sent to the <object> after the specified time has elapsed. The <message> is added to the <pendingMessages> <function>. The ID of the <message> is <return|returned> by the <result> <function>. To cancel the <message>, use the <cancel> <command> with this ID.
+The result: The ID of the <message> in the <pendingMessages> <function> queue.
```

Comment:
I think `The result:` description should still say that `the result` is the ID of the message in the <pendingMessages> queue if the `in <time>` form is used - how about 
```
The result: If you specify a <time>, <the result> is the ID of the <message> in the <pendingMessages> <function> queue.
```